### PR TITLE
Workflows main page refactoring - side bar

### PR DESCRIPTION
**Pull request by jozefbakus**
Implements: #563

### CODE REVIEW

File: src/main/scala/za/co/absa/hyperdrive/trigger/persistance/WorkflowRepository.scala [hunk 1]
```diff
@@ -46,7 +46,8 @@ trait WorkflowRepository extends Repository {
   def updateWorkflow(workflow: WorkflowJoined, user: String)(implicit ec: ExecutionContext): Future[Unit]
   def switchWorkflowActiveState(id: Long, user: String)(implicit ec: ExecutionContext): Future[Unit]
   def updateWorkflowsIsActive(ids: Seq[Long], isActiveNewValue: Boolean, user: String)(implicit ec: ExecutionContext): Future[Unit]
-  def getProjects()(implicit ec: ExecutionContext): Future[Seq[String]]
+  def getProjects()(implicit ec: ExecutionContext): Future[Seq[Project]]
+  def getProjectNames()(implicit ec: ExecutionContext): Future[Seq[String]]
   def getProjectsInfo()(implicit ec: ExecutionContext): Future[Seq[ProjectInfo]]
```
Comment (kevinwallimann) [inline src/main/scala/za/co/absa/hyperdrive/trigger/persistance/WorkflowRepository.scala:51]: Btw, is the `/workflows/projectsInfo` endpoint used anywhere?

File: ui/src/app/stores/workflows/workflows.effects.ts [hunk 2]
```diff
@@ -51,38 +51,41 @@ export class WorkflowsEffects {
     private jobService: JobService,
     private store: Store<AppState>,
     private router: Router,
-    private toastrService: ToastrService,
+    public toastrService: ToastrService,
```
Comment (kevinwallimann) [inline ui/src/app/stores/workflows/workflows.effects.ts]: Why is this `public`?

**Comment by kevinwallimann** [comment]
I tested the UI. Looks good

File: ui/src/app/stores/workflows/workflows.effects.ts (hunk 2 above)
Comment (jozefbakus) [reply ui/src/app/stores/workflows/workflows.effects.ts]: Oh s***.
I had a problem with "should return initialize workflows failure if workflowService.getJobTemplates responds with an error."
I tried to spy toastrService directly.
The problem was with the order.
I tested toaster call before I executed underTest.workflowsInitialize

File: src/main/scala/za/co/absa/hyperdrive/trigger/persistance/WorkflowRepository.scala (hunk 1 above)
Comment (jozefbakus) [reply src/main/scala/za/co/absa/hyperdrive/trigger/persistance/WorkflowRepository.scala:51]: Nope, I think the method itself was but I removed it.

Comment (jozefbakus) [reply src/main/scala/za/co/absa/hyperdrive/trigger/persistance/WorkflowRepository.scala:51]: I don't think so, we can remove it

Comment (kevinwallimann) [reply src/main/scala/za/co/absa/hyperdrive/trigger/persistance/WorkflowRepository.scala:51]: Ok, let's do it in another PR. Doesn't really have to do anything with this task

**Review by kevinwallimann — APPROVED** [review]

**Pull request merged**